src/printing.cpp

```
@@ -725,6 +727,17 @@ void print_map_descriptors(const std::vector<EbpfMapDescriptor>& descriptors, st
     }
 }
 
+bool disassemble(const RawProgram& raw_prog, const ebpf_verifier_options_t& options, std::ostream& out,
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)